index.js

```diff
@@ -0,0 +1,124 @@
+const java_version = [
+  "openjdk8",
+  "openjdk9",
+  "openjdk10",
+  "openjdk11"
```

Comment:
Should we pre add 12?  Or can we ignore the version number even?  Else if we forge to update this.

Reply:
Can't add jdk12 yet. It returns a 404 atm as there are no releases.